src/components/Header/TopNav/LogoLink.js

```diff
@@ -0,0 +1,29 @@
+import React, { Fragment } from 'react'
+import { Link } from 'gatsby'
+
+import logo from '../../../images/logo_animated.gif'
+import ServiceSpecialityLogoLink from '../../../images/service-speciality-logo-link'
+import { servicesRegExp } from './ServicesSpecialitiesMap'
+
+const LogoLink = ({ path = '/' }) => {
+  const isServicePage = path.search(servicesRegExp) > -1
+  const isSpecialityPage = path.includes('speciality')
+
+  return (
+    <Fragment>
+      {isSpecialityPage || isServicePage ? (
+        <ServiceSpecialityLogoLink
+          backgroundColor={isSpecialityPage ? '#52FFAC' : null}
+          textColor={isSpecialityPage ? '#090329' : null}
+          hoverBackgroundColor={isSpecialityPage ? '#FFFFFF' : null}
```

Comment:
I think you can pass only fillColor as a parameter , and keep the logic of onMouseEnter inside here, and make ServiceSpecialityLogoLink only `svg` that does not handle any logic, you won't need both `backgroundColor` and `hoverBackgroundColor` here.

Reply:
- [ ] done. The svg doesn't handle any logic now :)